internal/controllers/reconciliation/controller.go

```diff
@@ -50,8 +46,8 @@ type Controller struct {
 	timeout               time.Duration
 	readinessPollInterval time.Duration
 	upstreamClient        client.Client
-	discovery             *discovery.Cache
 	minReconcileInterval  time.Duration
+	disableSSA            bool
```

Comment:
I assume we'll set this on the overlay reconciler for old cluster versions right?

Reply:
Yeah exactly